README.md

```diff
@@ -40,6 +40,11 @@ These libraries contain helper functions, generalized standards, and other tools
 - [Unsigned Fixed Point Number](./libs/fixed_point/) is an interface to implement fixed-point numbers.
 - [StorageMapVec](./libs/storagemapvec/) is a temporary workaround for a StorageMap<K, StorageVec<V>> type.
 
+### Standards
+
+- [Token Standard](./standards/frc20/) is a standard ABI used to implement native assets on Fuel.
```

Comment:
Msybe this could say something like "this is used to add metadata to a native asset", as native assets need no work to implement at a most basic level.